apps/examples/testcase/le_tc/kernel/tc_libc_misc.c

```diff
@@ -347,6 +349,72 @@ static void tc_libc_misc_vdbg(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+ * @fn                  :tc_libc_misc_dumpbuffer
+ * @brief               :Do a pretty buffer dump
+ * @scenario            :Do a pretty buffer dump
+ * @API's covered       :lib_dumpbuffer
+ * @Preconditions       :None
+ * @Postconditions      :None
+ * @Return              :void
+ */
+static void tc_libc_misc_dumpbuffer(void)
+{
+	const char *msg = "tc_libc_misc_dumpbuffer";
+	unsigned char buffer[] = { 'S', 'A', 'M', 'S', 'U', 'N', 'G', '-', 'T', 'i', 'z', 'e', 'n', 'R', 'T' };
+	unsigned char *buf = NULL;
+	int idx;
+
+	/**
+	 * As lib_dumpbuffer returns void, there in no way to check the success or failure case
+	 * we need to manually check the dump output
+	 * Case-1: output: 53414d53554e472d54697a656e5254 SAMSUNG-TizenRT
+	 * Case-2: output: 0a0a0a0a0a0a0a0a0a0a0a0a0a0a0a ............... (i.e, 256 "0a")
+	 */
+
+	/* Case-1 */
+	lib_dumpbuffer(msg, buffer, sizeof(buffer));
+
+	/* Case-2 */
+	buf = (unsigned char *)malloc(BUFF_SIZE);
+	TC_ASSERT("malloc", buf);
+
+	for (idx = 0; idx < BUFF_SIZE; idx++) {
+		buf[idx] = 0xA;
+	}
+	lib_dumpbuffer(msg, buf, BUFF_SIZE);
+
+	free(buf);
+
+	TC_SUCCESS_RESULT();
+}
+
+/**
+ * @fn                  :tc_libc_misc_match
+ * @brief               :Simple shell-style filename pattern matcher
+ * @scenario            :Returns 1 (match) or 0 (no-match)
+ * @API's covered       :lib_match
+ * @Preconditions       :None
+ * @Postconditions      :None
+ * @Return              :void
+ */
+static void tc_libc_misc_match(void)
+{
+	const char *pattern = "?|";
+	const char *string = "S";
+	int ret;
```

Comment:
Two words, "?l" and "S" are assigned to const region becasue you used const keyword.
so these will not disappear after this function is finished.
And we can't access these values after we assign new value to pattern and string variable in line 410

Reply:
Yes, I agree with your point that 2 string literals will be there in read-only data section, even after function is finished.
1. Why do you want to access used string literals?
2. If your concern is about using string literals, I can provided better approach to that using char array.

Please elaborate your concern/question regarding this comment.

Reply:
Sorry, I misundersood for a moment. you don't need to fix it